tests/functional/iac/test_iac_scan_ci.py

```diff
@@ -0,0 +1,95 @@
+from pathlib import Path
+
+from tests.conftest import _IAC_SINGLE_VULNERABILITY
+from tests.functional.utils import run_ggshield_iac_scan
+from tests.repository import Repository
+
+
+def test_ci_diff_no_vuln(tmp_path: Path) -> None:
```

Comment:
Sorry, I missed that on the first review but this could be factorized and parametrized with `test_ci_all_no_vuln`

Reply:
Same for the tests with vulns.

Eventually, we may want to factorize the four tests into one.